lib/absinthe/lexer.ex

```diff
@@ -340,8 +338,8 @@ defmodule Absinthe.Lexer do
     {[chars], Map.put(context, :token_location, line_and_column(loc, byte_offset, 1))}
   end
 
-  defp mark_block_string_start(_rest, chars, context, loc, byte_offset) do
-    {[chars], Map.put(context, :token_location, line_and_column(loc, byte_offset, 3))}
+  defp mark_block_string_start(_rest, _chars, context, loc, byte_offset) do
```

Comment:
After changing the above two tests failed: "basic document, multiple lines with block string" and the ""document with emojis". This was because of usage of multiple lines with block string. 
It was because the first value (so `[[]]`) of the returned tuple of this function was a nested empty list instead of being an empty list.